src/altinn-app-frontend/src/components/base/ListComponent.tsx

```diff
@@ -127,31 +118,59 @@ export const ListComponent = ({
       }),
     );
   };
-  //Find the rowDataValue with the right dataModelBindings
-  const FindRowDataValue = (datalist) => {
+  const rowAsValue = (datalist) => {
     const chosenRowData: rowValue = {};
     for (const key in dataModelBindings) {
       chosenRowData[key] = datalist[key];
     }
     return chosenRowData;
   };
+  const rowAsValueString = (datalist) => {
+    const chosenRowData: rowValue = {};
+    for (const key in dataModelBindings) {
+      chosenRowData[key] = datalist[key];
+    }
+    return JSON.stringify(chosenRowData);
+  };
+
+  const createLabelRadioButton = (datalist) => {
+    let label = '';
+    for (const key in formData) {
+      label += `${key} ${datalist[key]} `;
+    }
+    return label;
+  };
 
   return (
     <Table
       selectRows={true}
       onChange={handleChange}
-      selectedValue={valueSelected}
+      selectedValue={formData as RowData}
     >
       <TableHeader>
-        <TableRow>{renderHeaders(tableHeaders)}</TableRow>
+        <TableRow>
+          <TableCell radiobutton={true}></TableCell>
+          {renderHeaders(tableHeaders)}
+        </TableRow>
       </TableHeader>
       <TableBody>
         {calculatedDataList.map((datalist) => {
           return (
             <TableRow
-              key={datalist}
-              rowData={FindRowDataValue(datalist)}
+              key={JSON.stringify(datalist)}
+              rowData={rowAsValue(datalist)}
             >
+              <TableCell radiobutton={true}>
+                <RadioButton
+                  name={datalist}
+                  // eslint-disable-next-line @typescript-eslint/no-empty-function
+                  onChange={() => {}}
```

Comment:
I think this should work, without having to disable eslint?
```suggestion
                  onChange={() => {
                    // Intentionally empty to prevent double-selection
                  }}
```